tests/cli_test.py

```diff
@@ -779,6 +792,8 @@ def test_configure_command_cannot_create_vdir(runner):
     )
     assert 'Exiting' in result.output
     assert result.exit_code == 1
+    # make sure pytest can clean up behind us
+    cleanup([runner.xdg_config_home, runner.xdg_data_home])
```

Comment:
Do we need to cleanup `xdg_config_home`? We didn't change it in this test.

Reply:
removed